app/models/aggregations.rb

```diff
@@ -81,4 +85,77 @@ def self.subjects
       }
     }
   end
+
+  def self.places
+    {
+      nested: {
+        path: 'subjects'
+      },
+      aggs: {
+        only_spatial: {
+          filter: {
+            terms: {
+              'subjects.kind': [
+                'Dublin Core; Spatial'
+              ]
+            }
+          },
+          aggs: {
+            place_names: {
+              terms: {
+                field: 'subjects.value.keyword'
+              }
+            }
+          }
+        }
+      }
+    }
+  end
+
+  def self.dates_ranges
+    {
+      nested: {
+        path: 'dates'
+      },
+      aggs: {
+        years: {
+          date_range: {
+            field: 'dates.value.as_date',
+            format: 'yyyy',
+            missing: '1989',
+            ranges: [
+              {
+                to: '1990'
+              },
+              {
+                from: '1990',
+                to: '2000'
+              },
+              {
+                from: '2000',
+                to: '2005'
+              },
+              {
+                from: '2005',
+                to: '2010'
+              },
+              {
+                from: '2010',
+                to: '2015'
+              },
+              {
+                from: '2015',
+                to: 'now/y'
```

Comment:
yeah it sucks honestly like this. Ultimately we haven't added data in so long that these buckets are fairly even in terms of number of results (for MIT GIS data only). I'm definitely open to making a change as this is TIMDEX API wide and not just for GIS. Will adjust.

Reply:
Noting that I requested a review before adjusting the dates. I'm not sure if I have a great solution yet so I might suggest we land this as-is (as awful as it is) and open a ticket to address improvements. I think we need to start with the UI and design what we want to display and then figure out the best way to achieve that rather than this approach fwiw.

Reply:
Dates filtering is being moved to a separate ticket.